### DISCUSSION TITLE
Seed Control

### DISCUSSION THREAD

**Discussion by JohnSabol55** [TinyTerra/ComfyUI_tinyterraNodes#128] [Q&A] [answered]
I would like to ask about Seed Control. 

I often generate pictures until I find one I like, then I activate the Save Image node, fix the seed number and regenerate the picture so that it will be saved. PipeKSamplerSDXL, after fixing the seed number, randomizes the seed once again, which changes the picture. So, the one picture I wanted to save is now gone.

Is there a way to fix the seed at the current number and regenerate the picture so that it can be saved?

I noticed there's a Seed function called "Control before generate" on the first picture on your page. Does it work like I described above? If yes, how can I activate it? I have tried switching it on but unfortunately with no success.

Thank you very much for your answers in advance.

**Comment by TinyTerra** [comment, marked as answer]
Control before generate is indeed what you're looking for. Personally I don't understand why after is the default in comfy. It makes less intuitive sense.

There is a setting in the main comfy settings called 'widget control mode' which you can set to before.
